src/ONYXKEYS.ts

```diff
@@ -1,5 +1,6 @@
 import type {OnyxEntry} from 'react-native-onyx';
 import type {ValueOf} from 'type-fest';
+import type * as FormTypes from './components/Form/inputs';
```

Comment:
I think it would make more sense to have these types in a different location like `src/types/form`, like we have for Onyx types.